ChangeLog.md

```diff
@@ -1,11 +1,105 @@
+      `https://account/container/virtual?SAS` and 
+      `https://account/container/virtualDir/*?SAS`.  The former copies the virtual directory
+      `virtualDir` by creating a folder of that name at the destination.  The latter copies the
+      _contents_ of `virtual` dir directly into the target without creating a folder named "virtualDir".
+   1. If you need to refer a literal `*` in the name of a blob, e.g. for a blob named "*", escape it with a backslash as `\*`
+1. The `--include` and `--exclude` parameters have been replaced by `--include-pattern` and
+   `--exclude-pattern` (for filenames) and `--include-path` and `--exclude-path` (for paths,
+   including directory and filenames).
+   The new parameters have behaviour that is better defined in complex situations (such as
+   recursion).  The `*` wildcard is supported in the pattern parameters, but _not_ in the path ones.
+1. There have been two breaking changes to the JSON output that is produced if you request
+   JSON-formatted output. The `sync` command's output in JSON has changed for consistency reasons,
+   and the final message type, for `copy` and `sync` has changed its name from `Exit` to `EndOfJob`.
+   Tools using the JSON output format to integrate AzCopy should be aware.
+1. If downloading to "null" on Windows the target must now be named "NUL", according to standard
+   Windows conventions.  "/dev/null" remains correct on Linux. (This feature can be used to test
+   throughput or check MD5s without saving the downloaded data.) 
+1. The file format of the (still undocmented) `--list-of-files` parameter is changed.  (It remains
+   undocmented because, for simplicity, users are
+   encouraged to use the new `--include-pattern` and `--include-path` parameters instead.)
+
+### New features
+
+1. `sync` is supported from Blob Storage to Blob Storage, and from Azure Files to Azure Files.
+1. `copy` is supported from Azure Files to Azure Files, and (TBC?) from Blob Storage to Azure Files.
+1. Percent complete is displayed as each job runs.
+1. VHD files are auto-detected as page blobs.
+1. A new benchmark mode allows quick and easy performance benchmarking of your network connection to
+   Blob Storage. Run AzCopy with the paramaters `bench --help` for details.  This feature is in
+   Preview status.
+1. The location for AzCopy's "plan" files can be specified with the environment variable
+   `AZCOPY_JOB_PLAN_LOCATION`.
+1. Log files and plan files can be cleaned up to save disk space, using AzCopy's new `jobs rm` and
+   `jobs clean` commands.
+1. When listing jobs with `jobs show`, the status of each job is included in the output.   
+1. The `--overwrite` parameter now supports the value of "prompt" to prompt the user on a
+   file-by-file basis. (The old values of true and false are also supported.)   
+1. The environment variable `AZCOPY_CONCURRENCY_VALUE` can now be set to "AUTO". This is expected to be
```

Comment:
Have we documented this in our environment variable description?

Reply:
Probably not. Since when I created it I wasn't sure it was the right long term approach (eg. maybe a range is better) and I didn't realise there would be so many support incidents coming through that it could address.

Reply:
Am still not sure I want to got out of my way to encourage it - at least not until its had a bit more bake time.